draft-ietf-ppm-dap.md

```diff
@@ -191,81 +191,83 @@ seen in the clear by any server.
+
+Aggregation parameter:
+: Parameter used to prepare a set of measurements for aggregation (e.g., the
+  candidate prefixes for Poplar1 from {{Section 8 of !VDAF}}). As defined in
+  {{!VDAF}}.
 
 Aggregator:
-: An endpoint that runs the input-validation protocol and accumulates input
-  shares.
+: An endpoint that recieves input shares from Clients and validates and
+  aggregates them with the help of the other Aggregators. As defined in
+  {{!VDAF}}.
 
 Batch:
-: A set of reports that are aggregated into an output.
+: A set of reports that are aggregated into an aggregate result.
 
 Batch duration:
 : The time difference between the oldest and newest report in a batch.
 
 Batch interval:
-: A parameter of the collect or aggregate-share request that specifies the time
-  range of the reports in the batch.
+: A parameter of a query issued by the Collector that specifies the time range
+  of the reports in the batch.
 
 Client:
 : The endpoint from which a user sends data to be aggregated, e.g., a web
-  browser.
+  browser. As defined in {{!VDAF}}.
 
 Collector:
-: The endpoint that receives the output of the aggregation function.
+: The endpoint that selects the aggregation parameter and receives the
+  aggregate result. As defined in {{!VDAF}}.
 
 Helper:
-: Executes the protocol as instructed by the leader.
-
-Input:
-: The measurement (or measurements) emitted by a client, before any encryption
-  or secret sharing scheme is applied.
+: An Aggregator that executes the aggregation and collection sub-protocols as
+  instructed by the leader.
 
 Input share:
-: An aggregator's share of the output of the VDAF
-  {{!VDAF=I-D.draft-irtf-cfrg-vdaf-03}} sharding algorithm. This algorithm is
-  run by each client in order to cryptographically protect its measurement.
+: An Aggregator's share of a measurement. The input shares are output by the
+  VDAF sharding algorithm. As defined in {{!VDAF}}.
 
-Leader:
-: A distinguished aggregator that coordinates input validation and data
-   collection.
+Output share:
+: An aggregator's share of the prepared measurement resulting from successful
+  execution of the VDAF preparation phase. Many output shares are combined into
+  an aggregate share during the VDAF aggregation phase. As defined in
+  {{!VDAF}}.
 
 Measurement:
-: A single value (e.g., a count) being reported by a client. Multiple
-  measurements may be grouped into a single protocol input.
+: A plaintext input emitted by a Client (e.g., a count, summand, or string),
+  before any encryption or secret sharing is applied. Depending on the VDAF in
+  use, multiple valus may be grouped into a single measurement. As defined in
+  {{!VDAF}}.
 
 Minimum batch duration:
-: The minimum batch duration permitted for a DAP task, i.e., the minimum time
-  difference between the oldest and newest report in a batch.
+: For a given DAP task, the minimum time difference between the oldest and
+  newest report in a batch.
```

Comment:
I think this is incorrect, because the actual times of reports doesn't affect the validity of a batch, only the definition of the batch interval. (In the most common case, batches will consist of a set of reports with the same client timestamp, due to client-side rounding. Such a batch would be valid, because the collector's query spanned the right amount of time, even though the time difference between the oldest and newest report is zero.) Moreover, this was renamed to "time precision" elsewhere in the document, so we should update and realphabetize this glossary entry to match.

Reply:
Removed minimum batch duration, since it's out of date. I don't think we need to define time precision here, since it's pretty technical.